gcloudutils/clients.go

```diff
@@ -0,0 +1,20 @@
+package gcloudutils
+
+import (
+	"context"
+
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/cloudbuild/v1"
+)
+
+func NewCloudBuildClient(ctx context.Context) (*cloudbuild.Service, error) {
```

Comment:
This code doesn't allow for selecting any other set of credentials than the default, just want to make sure that is intended.